src/domain/order/use-cases/get-orders.ts

```diff
@@ -0,0 +1,19 @@
+// import { NotFoundError } from '@cloud-burger/handlers';
+import logger from '@cloud-burger/logger';
+import { Order } from '../entities/order';
+import { OrderRepository } from '../repositories/order';
+
+export class GetOrdersUseCase {
```

Comment:
```suggestion
export class ListOrdersUseCase {
```

Eu mudaria o nome desse para List